task_sdk/src/airflow/sdk/log.py

```diff
@@ -159,153 +151,128 @@ def logging_processors(
         httpcore,
         httpx,
     )
+    return timestamper, processors, suppress
 
-    if enable_pretty_log:
-        rich_exc_formatter = structlog.dev.RichTracebackFormatter(
-            # These values are picked somewhat arbitrarily to produce useful-but-compact tracebacks. If
-            # we ever need to change these then they should be configurable.
-            extra_lines=0,
-            max_frames=30,
-            indent_guides=False,
-            suppress=suppress,
-        )
-        my_styles = structlog.dev.ConsoleRenderer.get_default_level_styles()
-        my_styles["debug"] = structlog.dev.CYAN
 
-        console = structlog.dev.ConsoleRenderer(
-            exception_formatter=rich_exc_formatter, level_styles=my_styles
-        )
-        processors.append(console)
-        return processors, {
-            "timestamper": timestamper,
-            "console": console,
-        }
+@cache
+def logging_processors_pretty():
+    timestamper, processors, suppress = _common_processors(timestamp_fmt="%Y-%m-%d %H:%M:%S.%f")
+
+    rich_exc_formatter = structlog.dev.RichTracebackFormatter(
+        # These values are picked somewhat arbitrarily to produce useful-but-compact tracebacks. If
+        # we ever need to change these then they should be configurable.
+        extra_lines=0,
+        max_frames=30,
+        indent_guides=False,
+        suppress=suppress,
+    )
+    my_styles = structlog.dev.ConsoleRenderer.get_default_level_styles()
+    my_styles["debug"] = structlog.dev.CYAN
+
+    console_renderer = structlog.dev.ConsoleRenderer(
+        exception_formatter=rich_exc_formatter, level_styles=my_styles
+    )
+    processors.append(console_renderer)
+    return processors, timestamper, console_renderer
+
+
+@cache
+def logging_processors_ugly():
```

Comment:
Maybe plain instead?

Reply:
Yeah just went with opposite of pretty :)

Reply:
+1 to plain

Reply:
done